app/scripts/controllers/ticketFinder.ctrl.js

```diff
@@ -61,14 +61,6 @@ angular.module('yourStlCourts').controller('TicketFinderCtrl', function (TicketF
     }
   };
 
-  function numberOfMunicipalitiesSelectedIsValid(){
```

Comment:
Why did you remove this function?

Reply:
This function was not used anywhere. The IDE showed me that easily